src/main/java/net/wasdev/wlp/ant/AbstractTask.java

```diff
@@ -327,16 +327,45 @@ public String waitForStringInLog(String regexp, long timeout,
      * @return The first line which includes the pattern, or null if the pattern
      *         isn't found or if the file doesn't exist
      */
-    protected String findStringInFile(String regexp, File fileToSearch) {
+    public String findStringInFile(String regexp, File fileToSearch) {
```

Comment:
Why was this changed from `protected` to `public`?

Reply:
We want to make use of that in https://github.com/OpenLiberty/ci.common/pull/63/files to get the contents of a string, without waiting and without having to count the number lines.